### ISSUE TITLE
Issue with parsing date format NA:-48.7 in 2.0.6?

### ISSUE THREAD

**Issue by mkweskin** [Cibiv/IQ-TREE#158]
When using `--date FILE` option to enter dates of ancestral nodes, the format of `NA:-48.7` gives an IQTREE error when run in IQTREE 2.0.6:

Date file:
```
taxa1,taxa2 NA:-48.7
```

Error:
```
Reading date file test-dates.txt ...
ERROR: Line 1: Expecting floating-point number, but found "NA:-48.7" instead
```

- Using 2.0.4, the same date file **is** parsed by IQTREE and passed to lsd.
- The format `-48.7:NA` can be parsed.
- I tried the example date in the manual, `NA:2018-02-01` and it also failed with the same error.

**Comment by mkweskin** [comment]
@bqminh replied to the IQTREE Google Group that he has fixed this and it will be incorporated into the next release: https://groups.google.com/d/msg/iqtree/QgE5KHmqOpo/u2Z3y3YyAgAJ

**Comment by bqminh** [comment]
Done, fixed in v2.1.1
